src/Essentials/src/AppInfo/AppInfo.android.cs

```diff
@@ -46,14 +47,13 @@ public void ShowSettingsUI()
 			context.StartActivity(settingsIntent);
 		}
 
-		static AppTheme GetRequestedTheme() => (Application.Context.Resources.Configuration.UiMode & UiMode.NightMask) switch
-		{
-			UiMode.NightYes => AppTheme.Dark,
-			UiMode.NightNo => AppTheme.Light,
-			_ => AppTheme.Unspecified
-		};
-
-		public AppTheme RequestedTheme => _requestedTheme.Value;
+		public AppTheme RequestedTheme
+			=> (Application.Context.Resources.Configuration.UiMode & UiMode.NightMask) switch
```

Comment:
We can at least cache the `Application.Context.Resources.Configuration` call here, right?

Reply:
I think it changes tbh:

> Return the **current configuration** that is in effect for this resource object. The returned object should be treated as read-only.

https://developer.android.com/reference/android/content/res/Resources#getConfiguration()

I also see https://developer.android.com/reference/android/content/res/Resources#updateConfiguration(android.content.res.Configuration,%20android.util.DisplayMetrics)

Reply:
I moved the caching to Controls as it is not needed in Core or Essentials.